server/src/main/java/com/google/musicanalysis/site/OAuthCallbackServlet.java

```diff
@@ -37,40 +37,51 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
     }
 
     var sessionOauthState = req.getSession().getAttribute("oauth:state");
+    var sessionOauthService = req.getSession().getAttribute("oauth:service");
```

Comment:
nit: you should only use `var` when the variable's type is explicitly declared when being initialized.

Ex:
```
var grid = new ArrayList<ArrayList<String>>();
```